sdk/tables/azure-tables/src/utils/bufferSerializer.browser.ts

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+export function isUint8Array(value: any): boolean {
```

Comment:
```suggestion
export function isUint8Array(value: any): value is Uint8Array {
```

This will actually narrow the type